decidim-initiatives/config/locales/en.yml

```diff
@@ -355,6 +356,7 @@ en:
         fill_data:
           back: Back
           continue: Continue
+          decidim_user_group_id_help: It's not possible to change initiative authorship after creation
           fill_data_help: "<ul> <li>Review the content of your initiative. Is your title easy to understand? Is the objective of your initiative clear?</li> <li>You have to choose the type of signature. In-person, online or a combination of both</li> <li>Which is the geographic scope of the initiative?</li> </ul>"
           initiative_type: Initiative type
```

Comment:
There's a CI error from i18n-tasks to remove this as is unused: `bundle exec i18n-tasks unused --locales en`
Can you drop this? Thanks
```suggestion
```